src/Processing.js

```diff
@@ -4133,7 +4167,9 @@
     */
     p.exit = function() {
       // cleanup
-      window.clearInterval(looping);
+      // requestAnimationFrame edit
```

Comment:
comments explaining edits go stale pretty much immediately. The commit message for the PR will tell us what this change was for, so we don't need to put that in the code itself.